### PR TITLE
tests: Slow events on pad @ 700 lines coverage - test collaboration events! :)

### PR DESCRIPTION
**Pull request by JohnMcLear**
Alternative approach as Cypress wasn't functioning / things weren't stable/reliable.

Test for https://github.com/ether/etherpad-lite/issues/1763 and Closes https://github.com/ether/etherpad-lite/issues/1763

This will use 5 additional Sauce browsers but that's okay as we now have 10 available.

This test introduces Frontend capability for adding front end tests, currently it just supports one targeted padId but there is nothing stopping other tests operating very similarly and having coverage for collaboration events.  I will leave that others to tinker with :)

### CODE REVIEW

File: src/tests/frontend/specs/responsiveness.js [hunk 1]
```diff
@@ -1,91 +1,56 @@
...
-
-xdescribe('Responsiveness of Editor', function () {
+// Test for https://github.com/ether/etherpad-lite/issues/1763
+describe('Responsiveness of Editor', function () {
   // create a new pad before each test run
   beforeEach(function (cb) {
-    helper.newPad(cb);
-    this.timeout(6000);
+    helper.newPad(cb, 'TEST_PAD_collab');
+    this.timeout(1200000);
```
Comment (rhansen) [inline src/tests/frontend/specs/responsiveness.js:11]: Is 20 minutes intentional? Or did you mean 2 minutes?

File: .github/workflows/frontend-collab-tests.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    - name: Disable import/export rate limiting
+      run: |
+        sed -e '/^ *"importExportRateLimiting":/,/^ *\}/ s/"max":.*/"max": 0/' -i settings.json
+
+    - uses: saucelabs/sauce-connect-action@v1
+      with:
+        username: ${{ secrets.SAUCE_USERNAME }}
+        accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+        tunnelIdentifier: ${{ steps.sauce_strings.outputs.tunnel_id }}
```
Comment (rhansen) [inline .github/workflows/frontend-collab-tests.yml:43]: The PR description say 5 tunnels, but I only see one here. Where are the others?

File: src/tests/frontend/specs/responsiveness.js (hunk 1 above)
Comment (JohnMcLear) [reply src/tests/frontend/specs/responsiveness.js:11]: Intentional.  Takes a while to run!

File: .github/workflows/frontend-collab-tests.yml (hunk 2 above)
Comment (JohnMcLear) [reply .github/workflows/frontend-collab-tests.yml:43]: My bad.  5 browser clients.  That's just one tunnel and 5 VMs (or whatever sauce labs call them)

Comment (JohnMcLear) [reply .github/workflows/frontend-collab-tests.yml:43]: Resolved

**Pull request closed without merging**